homeassistant/components/light/hue.py

```diff
@@ -397,4 +408,25 @@ def device_state_attributes(self):
         attributes = {}
         if self.is_group:
             attributes[ATTR_IS_HUE_GROUP] = self.is_group
+        if self.gamut_typ:
+            attributes[ATTR_GAMUT_TYP] = self.gamut_typ
+        if self.gamut:
+            attributes[ATTR_GAMUT] = self.gamut
         return attributes
+
+    def get_gamut(self):
+        """Return the gamut information of the light."""
+        if self.is_group:
+            return ('None', None)
+
+        try:
+            light_spec = self.light.raw['capabilities']['control']
+            color_gamut_type = light_spec['colorgamuttype']
+            color_gamut = tuple([tuple(x) for x in light_spec['colorgamut']])
```

Comment:
@balloob sad in a previous post:
> aiohue is just a thin wrapper around the Hue API. It should not contain any conversion logic. It would be the right place to store model mapping to color gamuts.

So I assumed he didn't want it in the aiohue, but i would be perfectly happy moving this function to aiohue.
What do you prefer @balloob?